albumentations/augmentations/transforms.py

```diff
@@ -2471,12 +2473,24 @@ class GaussianBlur(Blur):
         uint8, float32
     """
 
-    def __init__(self, blur_limit=7, always_apply=False, p=0.5):
-        super(GaussianBlur, self).__init__(blur_limit, always_apply, p)
+    def __init__(self, blur_limit=7, min_max_sigma=None, always_apply=False, p=0.5):
+        super(GaussianBlur, self).__init__(always_apply, p)
+        self.blur_limit = to_tuple(blur_limit, 3)
+        self.min_max_sigma = min_max_sigma
 
-    def apply(self, image, ksize=3, **params):
+    def apply(self, image, ksize=3, sigma=0, **params):
         return F.gaussian_blur(image, ksize)
 
+    def get_params(self):
+        if self.min_max_sigma is None:
+            return {"ksize": int(random.choice(np.arange(self.blur_limit[0], self.blur_limit[1] + 1, 2)))}
+        else:
+            sigma = random.uniform(self.min_max_sigma[0], self.min_max_sigma[1])
+            return {"sigma": sigma, "ksize": max(3, math.ceil((sigma - 0.5) / 0.3) * 2 + 1)}
```

Comment:
Please set `ksize` to zeros, `GaussianBlur` automatically produce `ksize` if sigma is set.

Reply:
Thanks for checking it guys, and thanks for the comment @Dipet 

Unfortunately, the c++ function always dies on an assert, if I set ksize=0:
`OpenCV(4.1.2) /io/opencv/modules/imgproc/src/smooth.dispatch.cpp:189: error: (-215:Assertion failed) ksize.width > 0 && ksize.width % 2 == 1 && ksize.height > 0 && ksize.height % 2 == 1 in function 'createGaussianKernels'`

looking at the code:
`ksize.width = cvRound(sigma1*(depth == CV_8U ? 3 : 4)*2 + 1)|1;`
https://github.com/opencv/opencv/blob/4.1.2/modules/imgproc/src/smooth.dispatch.cpp#L184
or master:
https://github.com/opencv/opencv/blame/master/modules/imgproc/src/smooth.dispatch.cpp#L291
I guess the depth is not right, because it dies with any sigma I use (even really large ones, like 10+)
Anyway, their formula would not work with sigmas below 0.5, as it gives you a useless ksize == 1

Reply:
OpenCV wait tuple of values `(0, 0)`. In this case OpenCV works well with any sigma value.

Reply:
@Dipet  gaussian_blur already transforms ksize into a tuple:
```
def gaussian_blur(img, ksize, sigma=0):
    # When sigma=0, it is computed as `sigma = 0.3*((ksize-1)*0.5 - 1) + 0.8`
    blur_fn = _maybe_process_in_chunks(cv2.GaussianBlur, ksize=(ksize, ksize), sigmaX=sigma)
    return blur_fn(img)
```

Do you guys approve the PR as it is, or should I do something else?